Runtime/Helpers/CinemachineSplineRollExtension.cs

```diff
@@ -0,0 +1,71 @@
+#if CINEMACHINE_UNITY_SPLINES
+using System;
+using UnityEngine;
+using UnityEngine.Splines;
+
+namespace Cinemachine
+{
+    /// <summary>
+    /// Extension that can be added to a SplineContainer or a vcam that uses a SplineContainer, for example a vcam
+    /// that has SplineDolly as Body component.
+    /// - When CinemachineSplineRollExtension is added to a gameObject that has SplineContainer, then any vcam that reads
+    /// that SplineContainer will be affected by the CinemachineSplineRollExtension overrides. It works as a global
+    /// override in this case.
+    /// - When CinemachineSplineRollExtension is added to a vcam, then the overrides are local, they only affect that
+    /// one vcam.
+    /// </summary>
+    [ExecuteInEditMode]
+    [DisallowMultipleComponent]
+    public class CinemachineSplineRollExtension : MonoBehaviour
```

Comment:
Can you just call this `CinemachineSplineRoll`?  I got confused with the current name and thought that it was a CinemachineExtension when it isn't.